db/convenience.cc

```diff
@@ -27,14 +27,14 @@ Status DeleteFilesInRange(DB* db, ColumnFamilyHandle* column_family,
       ->DeleteFilesInRange(column_family, begin, end);
 }
 
-Status VerifyChecksum(const std::string& file_path) {
+Status VerifySstFileChecksum(const Options& options,
+                             const EnvOptions& env_options,
+                             const std::string& file_path,
+                             const Comparator* comp) {
```

Comment:
If `Options` is passed, why do you still need to pass comparator? Also, it is too much ask for users to generate an option which covers both of DBOptions and CFOptions. I would prefer passing them separately.